DistroLauncher/messages.mc

```diff
@@ -95,3 +95,37 @@ Language=English
 Please enable the Virtual Machine Platform Windows feature and ensure virtualization is enabled in the BIOS.
 For information please visit https://aka.ms/enablevirtualization
 .
+
+MessageId=2001 SymbolicName=MSG_USAGE_EXTENDED
+Language=English
+Launches or configures a Linux distribution.
+
+Usage:
+    <no args>
+        Launches the user's default shell in the user's home directory.
+
+    install [options]
+        Install the distribution and do not launch the shell when complete.
+          --root
+              Do not create a user account and leave the default user set to root.
+          --autoinstall <AUTOINSTALL-FILE-PATH>
+              Reads information from an YAML file to automatically configure the distribution.
+          --enable-installer
```

Comment:
I think this should rather be something like `--gui` or `--oobe` as the command is `ubuntu.exe install <with something>`. Also, we should consider forcing or not TUI.

We should review all together the UX for this, but let’s not block on it for now.